test/models/apple/show_test.rb

```diff
@@ -256,7 +256,7 @@
 
     it "returns an authed url if private" do
       assert_equal apple_show.feed_published_url,
-        "https://p.prxu.org/#{public_feed.podcast.path}/#{public_feed.slug}/feed-rss.xml?auth=" + public_feed.tokens.first.token
+        "https://p.prxu.org/#{public_feed.podcast.path}/feed-rss.xml?auth=" + public_feed.tokens.first.token
```

Comment:
This test was a bit atypical - IRL you'd never have a `public_feed` that wasn't the `default_feed` (with a nil slug).